{"uid":"6c9d7414795cb6c0","name":"Test to verify that user can complete the pause task with exception","fullName":"tests.scenarios.job.test_job_execution.TestJobExecution#test_complete_pause_task_with_exception","historyId":"fbdea6d1e26b2a84c4966dfab8ece9c2","time":{"start":1733499021678,"stop":1733499021678,"duration":0},"description":"This testcase tests completing the pause task while executing the job","descriptionHtml":"<p>This testcase tests completing the pause task while executing the job</p>\n","status":"skipped","statusMessage":"Skipped: We cannot complete the task with exception when task are paused","statusTrace":"('/home/server1/Downloads/platform-api-develop/tests/scenarios/job/test_job_execution.py', 1422, 'Skipped: We cannot complete the task with exception when task are paused')","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"env_setup","time":{"start":1733490527832,"stop":1733490527832,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"login_fixture","time":{"start":1733490527832,"stop":1733490528079,"duration":247},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"get_facility","time":{"start":1733490528079,"stop":1733490528284,"duration":205},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"select_usecase_fixture","time":{"start":1733490528284,"stop":1733490528542,"duration":258},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"username_to_user_id_mapping","time":{"start":1733490528542,"stop":1733490529001,"duration":459},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"user_group_creation","time":{"start":1733498002570,"stop":1733498003483,"duration":913},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"flush_param_counters","time":{"start":1733499021680,"stop":1733499021681,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"flush_param_counters::0","time":{"start":1733499034920,"stop":1733499034920,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.skip(reason='We cannot complete the task with exception when task are paused')"},{"name":"tag","value":"complete_pause_task_with_exception"},{"name":"tag","value":"job"},{"name":"tag","value":"scenario"},{"name":"parentSuite","value":"tests.scenarios.job"},{"name":"suite","value":"test_job_execution"},{"name":"subSuite","value":"TestJobExecution"},{"name":"host","value":"ubuntu"},{"name":"thread","value":"29333-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.scenarios.job.test_job_execution"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"job_execution_by","value":"'user'"},{"name":"users_list","value":"['process.publisher.03']"}],"links":[{"name":"https://app.clickup.com/t/86cukreh9","url":"https://app.clickup.com/t/86cukreh9","type":"link"}],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["scenario","complete_pause_task_with_exception","job","@pytest.mark.skip(reason='We cannot complete the task with exception when task are paused')"]},"source":"6c9d7414795cb6c0.json","parameterValues":["'user'","['process.publisher.03']"]}